pkg/k8s/apis/cilium.io/v2alpha1/bgpp_types.go

```diff
@@ -122,18 +122,44 @@ func (gr *CiliumBGPNeighborGracefulRestart) SetDefaults() {
 // +kubebuilder:validation:Pattern=`^([0-9]|[1-9][0-9]{1,8}|[1-3][0-9]{9}|4[01][0-9]{8}|42[0-8][0-9]{7}|429[0-3][0-9]{6}|4294[0-8][0-9]{5}|42949[0-5][0-9]{4}|429496[0-6][0-9]{3}|4294967[01][0-9]{2}|42949672[0-8][0-9]|429496729[0-5])$|^([0-9]|[1-9][0-9]{1,3}|[1-5][0-9]{4}|6[0-4][0-9]{3}|65[0-4][0-9]{2}|655[0-2][0-9]|6553[0-5]):([0-9]|[1-9][0-9]{1,3}|[1-5][0-9]{4}|6[0-4][0-9]{3}|65[0-4][0-9]{2}|655[0-2][0-9]|6553[0-5])$`
 type BGPStandardCommunity string
 
+// BGPCommonCommunity type represents a value of commonly used "standard" 32-bit BGP Communities Attribute (RFC 1997)
+// as a text alias to their numeric value. Allowed values and their mapping to the numeric values:
+//
+//	internet                   = 0x00000000
```

Comment:
Maybe another tab for decimal representation (65535:<0-65535>) of these communities will be useful.